requirements.txt

```diff
@@ -11,3 +11,4 @@ Pillow==3.1.1
 requests==2.9.1
 ua-parser==0.6.1
 WTForms==2.1
+tldextract==1.7.5
```

Comment:
If tldextract is updated it presumably adds better data for the server name public prefix lists. I'd like to make the dependency >= so we get those updates.